hazelcast-jet-core/src/main/java/com/hazelcast/jet/Job.java

```diff
@@ -82,6 +83,12 @@ default String getName() {
     @Nonnull
     JobStatus getStatus();
 
+    /**
+     * Returns a copy of the current values of all job-specific metrics.
+     */
+    @Nonnull
+    Map<String, Long> getMetrics();
```

Comment:
it might be better to return some other class than Map, in case we wanted to add more information, such as timestamp.

Reply:
Done.